test/recipes/25-test_req.t

```diff
@@ -10,12 +10,14 @@
 use strict;
 use warnings;
 
+use POSIX;
```

Comment:
Maybe sufficient: `use POSIX qw(strftime);`

Reply:
You are right, this is better.
Also moved to `tconversion.pl`.